api/enrich.py

```diff
@@ -0,0 +1,119 @@
+from functools import partial
+
+import requests
+from datetime import datetime, timedelta
+from flask import Blueprint, current_app
+
+from api.schemas import ObservableSchema
+from api.utils import get_json, jsonify_data, jsonify_errors
+
+enrich_api = Blueprint('enrich', __name__)
+
+
+get_observables = partial(get_json, schema=ObservableSchema(many=True))
+
+
+def get_verdict(observable_value, observable_type,
+                disposition, valid_time):
+    source_url = current_app.config['API_URL'] \
+                 + current_app.config['API_SOURCE'].format(
+        observable=observable_value)
+
+    return {
+        'type': 'verdict',
+        'observable': {'type': observable_type, 'value': observable_value},
+        'disposition': disposition,
+        'valid_time': valid_time,
+        'judgement_id': source_url
```

Comment:
This isn't correct. This should be a transient ID of a judgment that led to the given verdict. And `source` and `source_uri` then have to be included in that particular judgment's payload.